controller/src/main/java/com/epam/esm/controller/app/Application.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.esm.controller.app;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+@SpringBootApplication(scanBasePackages = {"com.epam.esm.**"})
+@EntityScan("com.epam.esm.dao.model")
+
+public class Application {
```

Comment:
Why did you decide to put Application class here?

Reply:
Where should I put it instead?